worker/games.py

```diff
@@ -116,10 +118,10 @@ def verify_required_cutechess(cutechess):
     p.stdout.close()
 
     if p.returncode != 0:
-        raise Exception("Failed to find cutechess version info")
+        raise WorkerException("Failed to find cutechess version info")
```

Comment:
What about `FatalException` for `cutechess-cli` not meeting the version requirement?
In this case should be modified the comment.
```python
    # raise "FatalException". Currently this is only
    # done when the worker is too slow.
```

Reply:
Yes that is a good idea. I have not looked at the logic of how the cutechess binary is downloaded/installed.